internal/stage13.go

```diff
@@ -20,7 +20,7 @@ func testNumbers(stageHarness *test_case_harness.TestCaseHarness) error {
 		TestCases: []testcases.TestCase{
 			&testcases.TokenizeTestCase{FileContents: getRandIntAsString(), ExpectsError: false},
 			&testcases.TokenizeTestCase{FileContents: fmt.Sprintf("%d.%d", getRandInt(), getRandInt()), ExpectsError: false},
-			&testcases.TokenizeTestCase{FileContents: fmt.Sprintf("%d.0000", getRandInt()), ExpectsError: false},
+			&testcases.TokenizeTestCase{FileContents: fmt.Sprintf("%d%d.%d%d", getRandInt(), getRandInt(), getRandInt(), getRandInt()), ExpectsError: false},
```

Comment:
@andy1li hmm, the `.0` case is actually important for us to test since the user needs to print `.0` with exactly one `.0` at the end irrespective of how many zeroes were in the input. So I don't think this can be removed.

Maybe change the one above instead? `%d.%d` to `%d%d.%d%d`